R/BASiCS_CalculateSpikeIns.R

```diff
@@ -0,0 +1,23 @@
+#' Convert concentration in moles per microlitre to molecule counts
+#' @param ERCCConcentration A vector of concentrations, in moles per litre,
+#' of the spike-in genes in the mixture used.
+#' @param DilutionFactor The dilution factor applied to the spike-in mixture.
+#' e.g., 1μl per 50ml would be a 1/50000 \code{DilutionFactor}.
+#' @param VolumePerCell The volume of spike-in mixture added to each well, or to
+#' each cell.
+#' @return The molecule counts per well, or per cell, based on the input
+#' parameters.
+BASiCS_CalculateSpikeIns <- function(
```

Comment:
`ERCCConcentration` is an input for this function for not used within it. Moreover `ERCC_mmul` is not defined. 

This is the definition we had in the original workflow:

```
ERCC_mmul <- ERCC_conc$concentration.in. Mix.1.attomoles.ul. * (10ˆ(-18))
```

Are we going to have the ERCC concentration file (`ERCC_conc` in the workflow) pre-loaded in the library?

Reply:
I think my idea was we'd pass in the conc vector but this makes more sense